tests/parametric/test_tracer.py

```diff
@@ -43,6 +43,65 @@ def test_tracer_span_top_level_attributes(self, test_agent: _TestAgentAPI, test_
         assert child_span["meta"]["key"] == "val"
 
 
+@scenarios.parametric
+class Test_TracerSCITagging:
+    @missing_feature(context.library == "cpp", reason="FIXME: not implemented yet")
+    @missing_feature(context.library == "php", reason="FIXME: not implemented yet")
+    @missing_feature(context.library == "ruby", reason="FIXME: not implemented yet")
+    @missing_feature(context.library == "dotnet", reason="FIXME: different implementation")
```

Comment:
Does the current implemention differs from the RFC? if yes, it should be `@bug`

Other point, could you use manifest file to declare those `missing_feature`? If you don't know how to, ping me on slack, I'will help you.